wqflask/maintenance/gen_ind_genofiles.py

```diff
@@ -0,0 +1,117 @@
+# Example command: env GN2_PROFILE=/usr/local/guix-profiles/gn-latest-20220122 TMPDIR=/export/local/home/zas1024/gn2-zach/tmp WEBSERVER_MODE=DEBUG LOG_LEVEL=DEBUG SERVER_PORT=5002 GENENETWORK_FILES=/export/local/home/zas1024/gn2-zach/genotype_files SQL_URI=mysql://webqtlout:webqtlout@localhost/db_webqtl ./bin/genenetwork2 ./etc/default_settings.py -c ./maintenance/gen_ind_genofiles.py
```

Comment:
Ah yeah, this is easy for me to forget because I use an ultra-wide monitor. I'll see if I can add something to my IDE that will remind me (I think there's some setting for this).

I'll also just put that information in a docstring.